src/logsearch-config/src/logstash-filters/snippets/app.conf

```diff
@@ -81,10 +81,15 @@ if [@index_type] == "app" {
           rename => { "[parsed_json_field][origin]"      => "[@source][component]" }
         }
 
+        if ![parsed_json_field][event_type] {
```

Comment:
This changes unknown parsed fields to be `UnknownEvent` type. I think I like this.